python/src/UserDefined_doc.i.in

```diff
@@ -50,7 +50,19 @@ Create a 3d-distribution:
 ...         points[i, j] = 10 * (i + 1) + 0.1 * (j + 1)
 >>> distribution = ot.UserDefined(points, [0.3,0.2,0.25,0.25])
 >>> print(distribution)
-UserDefined({x = [10.1,10.2,10.3], p = 0.3}, {x = [20.1,20.2,20.3], p = 0.2}, {x = [30.1,30.2,30.3], p = 0.25}, {x = [40.1,40.2,40.3], p = 0.25})"
+UserDefined({x = [10.1,10.2,10.3], p = 0.3}, {x = [20.1,20.2,20.3], p = 0.2}, {x = [30.1,30.2,30.3], p = 0.25}, {x = [40.1,40.2,40.3], p = 0.25})
+
+Create :math:`\mathcal{U}\{0, 9\}`, the discrete uniform distribution on the set {0, 1, ..., 9}
+with equal weights:
+
+>>> points = [[i] for i in range(10)]
+>>> distribution = ot.UserDefined(points)
+
+Create :math:`\mathcal{U}\{9, 19\}`, the discrete uniform distribution on the set {9, 10, ..., 19}
+with equal weights:
+
+>>> points = [[i] for i in range(9, 20)]
+>>> distribution = ot.UserDefined(points)"
```

Comment:
these 2 are redundant, chose one

Reply:
I agree, and would choose the second one because it is a little more unusual.

Reply:
I agree. I kept the second, as suggested.